python/paddle/incubate/autograd/composite_rules.py

```diff
@@ -59,6 +59,13 @@ def composite_batchnorm(
     trainable_statistics,
 ):
     """define composite rule of op batch_norm"""
+    is_amp = False
+    from paddle.fluid.data_feeder import convert_dtype
+
+    if convert_dtype(x.dtype) == "float16":
```

Comment:
if not in amp case but x.shape = "float16", should we also do this?

Reply:
May not have this case, I think